### ISSUE TITLE
Get long message in Spyder when I use SymPy consoles

### ISSUE THREAD

**Issue by bob5280** [spyder-ide/spyder#14928]
### What steps will reproduce the problem?

Hello!  I used SymPy 1.0 in the Spyder Console and now on every input I get this long message: Output from spyder call 'get_namespace_view':
C:\Users\rwden\AppData\Local\Programs\Spyder\pkgs\sympy\deprecated\class_registry.py:38: SymPyDeprecationWarning: 

C, including its class ClassRegistry, has been deprecated since SymPy
1.0. It will be last supported in SymPy version 1.0. Use direct
imports from the defining module instead. See
https://github.com/sympy/sympy/issues/9371 for more info.

  deprecated_since_version='1.0').warn(stacklevel=2)
  
  
  
How can I prevent this message from appearing every time I do an input?  Thank you.  I am new to Spyder and it is great!

## Versions

* Spyder version: 4.2.3 
* Python version: 3.7.9
* Qt version: 5.12.10
* PyQt5 version: 5.12.3
* Operating System: Windows 10

### Dependencies

```

# Mandatory:
atomicwrites >=1.2.0            :  1.4.0 (OK)
chardet >=2.0.0                 :  4.0.0 (OK)
cloudpickle >=0.5.0             :  1.6.0 (OK)
diff_match_patch >=20181111     :  20200713 (OK)
intervaltree                    :  None (OK)
IPython >=7.6.0                 :  7.21.0 (OK)
jedi =0.17.2                    :  0.17.2 (OK)
jsonschema >=3.2.0              :  3.2.0 (OK)
keyring >=17.0.0                :  22.3.0 (OK)
nbconvert >=4.0                 :  6.0.7 (OK)
numpydoc >=0.6.0                :  1.1.0 (OK)
paramiko >=2.4.0                :  2.7.2 (OK)
parso =0.7.0                    :  0.7.0 (OK)
pexpect >=4.4.0                 :  4.8.0 (OK)
pickleshare >=0.4               :  0.7.5 (OK)
psutil >=5.3                    :  5.8.0 (OK)
pygments >=2.0                  :  2.8.0 (OK)
pylint >=1.0                    :  2.7.2 (OK)
pyls >=0.36.2;<1.0.0            :  0.36.2 (OK)
pyls_black >=0.4.6              :  0.4.6 (OK)
pyls_spyder >=0.3.2             :  0.3.2 (OK)
qdarkstyle >=2.8                :  2.8.1 (OK)
qtawesome >=0.5.7               :  1.0.2 (OK)
qtconsole >=5.0.1               :  5.0.2 (OK)
qtpy >=1.5.0                    :  1.9.0 (OK)
rtree >=0.8.3                   :  0.9.4 (OK)
setuptools >=39.0.0             :  54.0.0 (OK)
sphinx >=0.6.6                  :  3.5.1 (OK)
spyder_kernels >=1.10.2;<1.11.0 :  1.10.2 (OK)
textdistance >=4.2.0            :  4.2.1 (OK)
three_merge >=0.1.1             :  0.1.1 (OK)
watchdog                        :  1.0.2 (OK)
zmq >=17                        :  22.0.3 (OK)

# Optional:
cython >=0.21                   :  0.29.22 (OK)
matplotlib >=2.0.0              :  3.3.4 (OK)
numpy >=1.7                     :  1.19.3 (OK)
pandas >=1.1.1                  :  1.2.3 (OK)
scipy >=0.17.0                  :  1.6.1 (OK)
sympy >=0.7.3                   :  1.7.1 (OK)
```

**Comment by Dan-Patterson** [comment]
This will be added to https://github.com/sympy/sympy/wiki/Release-Notes-for-1.8.
If you can't wait for sympy 1.8 you can see if you can follow their instructions in your link

**Comment by appomsk** [comment]
The same question. _"How can I prevent this message from appearing every time I do an input? "_

**Comment by steff456** [comment]
Hi @bob5280, @Dan-Patterson and @appomsk

Can you guys please open a Sympy console and see if the message persists? You can open one in the hamburger menu of the console > New special console > New Sympy console.

Please let me know and thanks for reporting!

**Comment by appomsk** [comment]
Hi, @steff456 

Yes. When I open Sympy console I see:
```
Python 3.9.2 -- IPython 
Output from spyder call 'get_namespace_view':
C:\Users\yazu\AppData\Local\Programs\Python\Python39\lib\site-packages\sympy\deprecated\class_registry.py:33: SymPyDeprecationWarning: 

C, including its class ClassRegistry, has been deprecated since SymPy
1.0. It will be last supported in SymPy version 1.0. Use direct
imports from the defining module instead. See
https://github.com/sympy/sympy/issues/9371 for more info.

  SymPyDeprecationWarning(
```
And then this message appears after every any command.

My environment:

- Windows 10
- Python 3.9.2 (from python.org)
- Sympy 1.7.1 (installed  with pip)
- Spyder 4.2.3 (installed with windows installer)

`isympy` in windows terminal works without any problem.

**Comment by oscarbenjamin** [comment]
This is fixed in SymPy 1.8 which has been released some time ago. If you update to the latest SymPy with
```
pip install -U sympy
```
then you will not see this warning any more.

I'm not sure though given an issue like this what the best way is to deprecate something without warnings showing up from introspective tooling like in ipython.

**Comment by steff456** [comment]
@oscarbenjamin did you fixed your issue with the upgrade to SymPy 1.8?

**Comment by oscarbenjamin** [comment]
I haven't personally noticed this issue (I don't use Spyder) but as a maintainer of SymPy I can tell you that this warning will not be seen any more. The `ClassRegistry` and all associated code including the code that emits the DeprecationWarning in the OP was fully removed from the SymPy codebase in https://github.com/sympy/sympy/pull/20896
